### ISSUE TITLE
traceplot does not take 1 < number < all variables

### ISSUE THREAD

**Issue by gerkovink** [amices/ggmice#63]
``` r
library(mice)
library(ggmice)
imp <- mice(boys, m=2, maxit = 2, print=FALSE)
plot(imp, c("hgt", "wgt", "bmi"))
```

![](https://i.imgur.com/mwCOItk.png)

``` r
plot_trace(imp, c("hgt", "wgt", "bmi"))
#> Error in if (vrb %nin% varlist & vrb != "all") {: the condition has length > 1
```

<sup>Created on 2022-07-11 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

**Comment by rjverheijden** [comment]
I encountered the same problem. It could be fixed by simply changing:

-  `if (vrb %nin% varlist & vrb != "all") {` on [line 26](https://github.com/amices/ggmice/blob/864373d194c9e4ebe02b869fef190a07f8247b86/R/trace.R#L26) to `if (TRUE %in% (vrb %nin% varlist & vrb != "all")) {` and

- `if (vrb == "all") {` on [line 29](https://github.com/amices/ggmice/blob/864373d194c9e4ebe02b869fef190a07f8247b86/R/trace.R#L29) to `if (TRUE %in% (vrb == "all")) {`.

**Comment by hanneoberman** [comment]
Fixed in #80

**Comment by hanneoberman** [comment]
Fixed in #80

``` r
library(mice)
#> 
#> Attaching package: 'mice'
#> The following object is masked from 'package:stats':
#> 
#>     filter
#> The following objects are masked from 'package:base':
#> 
#>     cbind, rbind
library(ggmice)
#> 
#> Attaching package: 'ggmice'
#> The following objects are masked from 'package:mice':
#> 
#>     bwplot, densityplot, stripplot, xyplot
imp <- mice(boys, m=2, maxit = 2, print=FALSE)
plot(imp, c("hgt", "wgt", "bmi"))
```

![](https://i.imgur.com/NNw19d8.png)

``` r
plot_trace(imp, c("hgt", "wgt", "bmi"))
```

![](https://i.imgur.com/su7gF5S.png)

<sup>Created on 2023-06-27 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>
